.github/workflows/ci.yml

```diff
@@ -1,7 +1,104 @@
+########################################################################################################
+# Build configurations
+# ====================
+#
+# full
+# ----
+# Description: Basic build with all components (TLX, C++, Cython). Has to be tested on every fully 
+#              supported platform. Close to user-experience, except that warnings are also treated 
+#              as errors during build and TLX is provided as externally build library (normal 
+#              installation builds both NetworKit and TLX in one step).
+# Script:      scripts/full.sh
+# Build-type:  Release
+#
+# 
+# full (variant)
+# ---------------------
+# Description: These builds are variants of full with different options.
+# Script:      scripts/full.sh
+# Variants:    native = Build for native architecture
+# 
+#         
+# core
+# ----
+# Description: Builds the C++ code into one shared library. Has also sanity checks enabled.
```

Comment:
This description misses the rationale for `core`: the reason why we do these core builds is that they are faster than `full` builds, right?

Reply:
Yes, main argument is heaving more speed for Debug-builds. `full` is by default a Release-build, while `core` is a Debug-build. Added this information to the description.